test/profiler/test_profiler.py

```diff
@@ -1661,6 +1661,7 @@ def test_nested_tensor_with_shapes(self):
                 self.assertTrue(len(e.input_shapes[0]) > 0)
 
     @patch.dict(os.environ, {"KINETO_USE_DAEMON": "1"})
+    @patch.dict(os.environ, {"KINETO_DAEMON_INIT_DELAY_S": "1"})
```

Comment:
May be useful to include the unit, like `KINETO_DAEMON_INIT_DELAY_SEC`

Reply:
Ah didn't spell SEC but have the S in the end KINETO_DAEMON_INIT_DELAY_S